packages/shared/src/components/squads/SquadHeaderBar.tsx

```diff
@@ -45,48 +45,50 @@ export function SquadHeaderBar({
     });
 
   return (
-    <div className={classNames('flex flex-wrap gap-4', className)} {...props}>
-      <div className="flex items-center rounded-14 border border-theme-divider-secondary">
+    <div
+      {...props}
+      className={classNames(
+        'flex flex-row flex-wrap items-center gap-4',
+        className,
+      )}
+    >
+      <div className="flex overflow-hidden items-center rounded-14 border border-theme-divider-secondary">
         <SimpleTooltip placement="top" content="Members list">
-          <Button
-            className="active:bg-theme-active btn-tertiary !pl-1 !pr-3 !rounded-r-none"
-            buttonSize="medium"
+          <button
```

Comment:
Why the change?

Reply:
There were a lot of overrides on the base styles of the button itself which lost the purpose of using the component itself.